server/db/postgres/sequelize_config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  development: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_development',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  test: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_test',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  production: {
+    use_env_variable: 'POSTGRES_DB_URL'
+  }
```

Comment:
I will add those config settings to the production block too since sequelize preferentially uses the use_env_variable option, that way you can just comment that single line to test prod locally